### PR TITLE
CFM 1133: Add information fields to group title header.

### PR DESCRIPTION
**Pull request by anvmn**
https://issuetracker.amplexor.com/jira/browse/CFM-1133

### CODE REVIEW

**Comment by anvmn** [comment]
selection_045
selection_046
selection_047

**Comment by ordavidil** [comment]
@anvmn 
Please do not assign before travis tests are passed.

**Comment by anvmn** [comment]
@ordavidil 
All tests passed

File: project/profiles/capacity4more/modules/c4m/content/c4m_content_group/theme/c4m-content-group-header-title.tpl.php [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+?>
+<div class="row" id="content-group-header-title">
+  <div class="col-sm-12 col-md-8">
+    <h1><?php print $title;?></h1>
+  </div>
+  <div class="col-sm-12 col-md-4" id="content-group-header-title-right-div">
+    <div class="row">
+      <div class="col-xs-offset-7 col-xs-5">
+        <div class="top-buffer indication indication-type">
+          group
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/content/c4m_content_group/theme/c4m-content-group-header-title.tpl.php]: @soniCaH 
Is it ok to be hard coded over here ? or this should be a dynamic value ?

File: project/profiles/capacity4more/themes/c4m/kapablo/build/src/stylesheets/components/_header.scss [hunk 2]
```diff
@@ -155,3 +155,74 @@
     background-color: $color-gray-light;
   }
 }
+
```
Comment (ordavidil) [inline project/profiles/capacity4more/themes/c4m/kapablo/build/src/stylesheets/components/_header.scss]: remove this line.

File: project/profiles/capacity4more/modules/c4m/content/c4m_content_group/c4m_content_group.block.inc [hunk 3]
```diff
@@ -30,9 +30,24 @@ function _c4m_content_group_block_header_name_banner() {
   // First establish the current group.
   $group = c4m_og_current_group();
   $wrapper = entity_metadata_wrapper('node', $group);
+  if (!empty($wrapper->pluggable_node_access->value())) {
+    // If pluggable_node_access is set, then access is restricted.
+    $group_access = 'restricted';
+  }
+  else {
+    // Determine access per group_access parameter.
+    $group_access = $wrapper->group_access->value() == FALSE ? 'public' : 'private';
```
Comment (anvmn) [inline project/profiles/capacity4more/modules/c4m/content/c4m_content_group/c4m_content_group.block.inc]: `$group_access = $wrapper->group_access->value()  ?  'private' :  'public';`

File: project/profiles/capacity4more/modules/c4m/content/c4m_content_group/c4m_content_group.block.inc [hunk 4]
```diff
@@ -30,9 +30,24 @@ function _c4m_content_group_block_header_name_banner() {
...
+  }
 
   $title = $wrapper->title->value();
   $status = isset($wrapper->c4m_og_status) ? $wrapper->c4m_og_status->value() : 'published';
+  $variables = array(
+    'title' => $title,
```
Comment (anvmn) [inline project/profiles/capacity4more/modules/c4m/content/c4m_content_group/c4m_content_group.block.inc]: might need `check_plain()`

File: project/profiles/capacity4more/modules/c4m/content/c4m_content_group/theme/c4m-content-group-header-title.tpl.php (hunk 1 above)
Comment (soniCaH) [inline project/profiles/capacity4more/modules/c4m/content/c4m_content_group/theme/c4m-content-group-header-title.tpl.php]: OK for now. Projects have different theming, but might re-use the title. If so, we'll refactor here.

File: project/profiles/capacity4more/modules/c4m/content/c4m_content_group/theme/c4m-content-group-header-title.tpl.php [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+/**
+ * @file
+ * Template for presenting additional information fields.
+ *
+ * Type, Status and Access at content group header title.
+ */
+?>
+<div class="row" id="content-group-header-title">
+  <div class="col-sm-12 col-md-8">
+    <h1><?php print $title;?></h1>
```
Comment (anvmn) [inline project/profiles/capacity4more/modules/c4m/content/c4m_content_group/theme/c4m-content-group-header-title.tpl.php]: Remove h1

Comment (soniCaH) [inline project/profiles/capacity4more/modules/c4m/content/c4m_content_group/theme/c4m-content-group-header-title.tpl.php]: Problem is that this entire TPL is printed between the H1-tag (so all the divs as well)

Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/content/c4m_content_group/theme/c4m-content-group-header-title.tpl.php]: @soniCaH 
what @anvmn meant is that eventually this line will like that:
```PHP
<?php print $title; ?>
```

**Comment by soniCaH** [comment]
Merged into CFM-20 to finish.

**Pull request closed without merging**